laravel/routes/web.php

```diff
@@ -19,6 +19,10 @@
 
 Route::get('/order', 'StockController@order')->name('order');
 
+Route::get('/order', 'StockController@order')->name('order');
+
+Route::get('/ordersindex', 'StockController@order')->name('ordersindex');
```

Comment:
一般的に、サービス開發をする上でURLを決める際には
RESTful APIという規則に沿って作成していくのが定石です。

そして一覽ページの場合はURLに `index` と付け無いほうが良さそうです。
```suggestion
Route::get('/orders', 'StockController@order')->name('ordersindex');
```
RESTful APIについては実際にどんな物なのか調べてもらえればと思います🐱